README.md

```diff
@@ -93,6 +93,12 @@ $token->getRefreshToken();
 $token->getExpires();
 ```
 
+Auth tokens are valid for 5 minutes. If you want to cache your tokens for as much as possible, you will need to pass a [PSR-16](https://www.php-fig.org/psr/psr-16/) compartible cache implementation. Laravel, Symfony and all major frameworks are already compatible with this interface. To enable cache, you only need to call the `setCache` method passing a `Psr\SimpleCache\CacheInterface` implementation.
```

Comment:
"for as _long_ as possible"

"compatible" instead of "compartible"? Or maybe it was supposed to be "comparable"?

"To enable _the_ cache"